src/Life/Github.hs

```diff
@@ -92,34 +102,54 @@ withSynced action = insideRepo $ do
 -- File manipulation
 ----------------------------------------------------------------------------
 
+data CopyDirection = FromHomeToRepo | FromRepoToHome
+
+updateFromRepo :: Set (Path Rel File) -> Set (Path Rel Dir) -> IO ()
+updateFromRepo withoutFiles withoutDirs = insideRepo $ do
+    infoMessage "Copying files from repo to local machine..."
+
+    let excludeLife = LifeConfiguration withoutFiles withoutDirs
+    repoLife <- parseRepoLife
+    let lifeToLive = lifeConfigMinus repoLife excludeLife
```

Comment:
Maybe here I should also check that files from arguments are not in repo life config? User could have small typo which can ruin things... Like write this:

```shell
$ life pull ChShersh --no-file profile
```

instead of

```shell
$ life pull ChShersh --no-file .profile
```